### PR TITLE
Replace Akka with Pekko for classes and functions

### PR DESCRIPTION
**Pull request by mdedetrich**
Self explanatory

### CODE REVIEW

**Review by nvollmar — APPROVED** [review]
lgtm

**Review by jrudolph — APPROVED** [review]
lgtm

**Comment by mdedetrich** [comment]
@jrudolph You happy with me merging this now?

File: docs/src/test/java/docs/http/javadsl/server/PekkoHttp1020MigrationExample.java [hunk 1]
```diff
@@ -13,7 +13,7 @@
 import org.apache.pekko.stream.Materializer;
 
 @SuppressWarnings("deprecation")
-public class AkkaHttp1020MigrationExample {
+public class PekkoHttp1020MigrationExample {
```
Comment (spangaer) [inline docs/src/test/java/docs/http/javadsl/server/PekkoHttp1020MigrationExample.java:16]: Just accidentally stumbled over this, but seems like this one would correctly be named Akka, as there is no Pekko 10.2 Migration

Comment (pjfanning) [reply docs/src/test/java/docs/http/javadsl/server/PekkoHttp1020MigrationExample.java:16]: I vote for keeping the new name - it easier to explain all the names now have Pekko in them - as opposed to explaining on a case by case basis. One extra option is to just to remove this test.

Comment (spangaer) [reply docs/src/test/java/docs/http/javadsl/server/PekkoHttp1020MigrationExample.java:16]: I don't have any firm opinion, but unfirmly: removing it seems like the most right option then.

Comment (mdedetrich) [reply docs/src/test/java/docs/http/javadsl/server/PekkoHttp1020MigrationExample.java:16]: Its likely these will be removed as they don't make sense in Pekko, but trademarks (and other legal matters) generally takes precedence over other matters.

**Pull request merged**